accord-core/src/test/java/accord/impl/basic/DelayedCommandStores.java

```diff
@@ -278,6 +284,7 @@ public DelayedSafeStore(DelayedCommandStore commandStore, RangesForEpoch ranges,
         @Override
         public void postExecute()
         {
+            super.postExecute();
```

Comment:
This call would invalidate safe InMemorySafeCommand, which will make `safe.isModified` to return `false`, and we will skip persisting anything in the journal.

Reply:
good catch, looks like a bad (or brain-dead) merge